src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -679,6 +679,17 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
             });
         }
 
+        // Adjust the height of rows in the row_selectable and row_deletable columns
+        if (fixed_columns || (row_selectable || row_deletable)) {
+            Array.from(r1c1.querySelectorAll(`tr`)).forEach((tr, index) => {
+                const style = getComputedStyle(tr);
+                const height = style.height;
+
+                this.stylesheet.setRule(`.dash-fixed-column:not(.dash-fixed-row) tr:nth-of-type(${index + 1})`,
+                    `height: ${height} !important;`);
+            });
+        }
+
```

Comment:
Sadly that's probably the only we can do. Concerns on my side are mostly about performance.

Reply:
Tried the issue example with 
```
data = data * 4
```

For 568 rows, initial table rendering took ~25 seconds 😢 mostly in rendering / relayout